go.mod

```diff
@@ -8,28 +8,33 @@ require (
 	github.com/mitchellh/cli v1.1.5
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/sirupsen/logrus v1.9.3
+	golang.org/x/oauth2 v0.15.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
 	github.com/Masterminds/goutils v1.1.1 // indirect
-	github.com/Masterminds/semver/v3 v3.2.1 // indirect
```

Comment:
It looks like this PR is currently reverting many of the version upgrades from https://github.com/PagerDuty/go-pagerduty/pull/504. If we can get that sorted out then we can get this merged...